apps/namadillo/package.json

```diff
@@ -27,7 +27,7 @@
     "jotai": "^2.6.3",
     "jotai-tanstack-query": "^0.8.5",
     "lodash.debounce": "^4.0.8",
-    "namada-chain-registry": "https://github.com/anoma/namada-chain-registry",
+    "namada-chain-registry": "https://github.com/anoma/namada-chain-registry#commit=50cb587b8a7128a276ed282258bae21ca2549b50",
```

Comment:
Will revert once the chain registry PR is merged! https://github.com/anoma/namada-chain-registry/pull/26

Reply:
I think you can revert it already in case the yarn.lock is already updated :)